templates/components/engines/typescript/agent/chat.ts

```diff
@@ -28,7 +29,14 @@ export async function createChatEngine() {
     const config = JSON.parse(
       await fs.readFile(path.join("config", "tools.json"), "utf8"),
     );
-    tools = tools.concat(await ToolsFactory.createTools(config));
+
+    // add function tools
+    const functionTools = getFunctionTools(config.local);
+    tools.push(...functionTools);
```

Comment:
how about this naming?
```suggestion
    // add local tools from the 'tools' folder (if configured)
    const localTools = createLocalTools(config.local);
    tools.push(...localTools);
```